<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DifficultySelect</class>
 <widget class="QWidget" name="DifficultySelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="hardButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>460</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
   </property>
   <property name="text">
    <string>Hard</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QLabel" name="difficultyTitle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>781</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
   </property>
   <property name="text">
    <string>Difficulty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="mediumButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
   </property>
   <property name="text">
    <string>Medium</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
  </widget>
  <widget class="QPushButton" name="easyButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>500</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(0, 0, 0, 245);</string>
   </property>
   <property name="text">
    <string>Easy</string>
   </property>
   <property name="autoRepeatDelay">
    <number>300</number>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="gameresources.qrc">:/screenshots/background.png</pixmap>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>hardButton</zorder>
  <zorder>difficultyTitle</zorder>
  <zorder>mediumButton</zorder>
  <zorder>easyButton</zorder>
 </widget>
 <resources>
  <include location="gameresources.qrc"/>
 </resources>
 <connections/>
</ui>
